backend/CMakeLists.txt

```diff
@@ -16,3 +19,20 @@ add_executable(${PROJECT_NAME} ${SOURCES})
 target_link_libraries(${PROJECT_NAME} Crow::Crow SQLite::SQLite3)
 
 target_include_directories(${PROJECT_NAME} PRIVATE src)
+
+## tests
+project(tests)
+
+find_package(Catch2 REQUIRED)
+
+set (TEST_SOURCE
+    tests/test.cpp
+)
+
+add_executable(${PROJECT_NAME} ${TEST_SOURCE})
+
+target_link_libraries(${PROJECT_NAME} PRIVATE Catch2::Catch2WithMain)
+
+target_include_directories(${PROJECT_NAME} PRIVATE tests)
```

Comment:
**suggestion**: This line isn't necessary because there aren't any header files in `/tests`. If you want to include `passwords.hpp`, make this the `/src` folder.